src/popover/popover.scss

```diff
@@ -182,8 +188,40 @@
 }
 
 .Popover-message--large {
-
   @include breakpoint(sm) {
     min-width: 320px;
   }
 }
+
+// Responsive Popover
+// For < md it will show full-width anchored to the bottom
+
+@media (max-width: ($width-md - 1px)) {
+  .Popover {
+    position: fixed;
+    top: auto;
+    right: 0;
+    bottom: 0;
+    left: 0;
```

Comment:
@langermank Sorry this is too late and would need to be a follow-up PR.

But we might wanna keep the `!important`. There are cases where Popovers are positioned manually to adjust to certain content. E.g. using [`right: -327px; bottom: -8px`](https://github.com/github/github/blob/bd7636336ba69e232df0613e68bfd1f37d56aa89/app/views/user_statuses/_notice.html.erb#L3).

Also the `width` of the `Popover-message` that probably gets customized too. E.g. [`width: 250px;`](https://github.com/github/github/blob/bd7636336ba69e232df0613e68bfd1f37d56aa89/app/views/repos/advisories/_title.html.erb#L14)

Reply:
Ah! No problem, will follow up with a fix. Thanks :)